components/card/cardPage.css

```diff
@@ -0,0 +1,43 @@
+.container__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container__components-holder {
+    width: fit-content;
+    height: fit-content;
+    padding: 1.6rem;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--primary-color-p2-lgt-30pct);
+}
+
+.container__components-holder > div {
+    flex-basis: 0;
+    flex-grow: 1;
```

Comment:
short-hand can be used. just a suggestion

Reply:
Personal preference, I like it more explicit.